### ISSUE TITLE
Type of document for PubMatic's “Master Services Agreement”

### ISSUE THREAD

**Issue by clementbiron** [ambanum/OpenTermsArchive#268]
I am in the process of adding documents to follow suggested by the GDI (https://disinformationindex.org/)  and I cannot determine the type of one of them.

What do you think @MattiSG  is the type of this document : https://pubmatic.com/legal/publisher-master-services-agreement/ ?

**Comment by MattiSG** [comment]
- I understand that PubMatic is the _service provider_.
- Based on the colophon on the website: “PubMatic delivers superior revenue to publishers”, I understand that “publishers” are PubMatics’ _end users_.
- I understand that this document is an “Agreement” that “is entered into by and between PubMatic and the Publisher”, to define the conditions of the usage of PubMatic by the Publisher (provision, term, warranties, costs, payments…).

To recap: this document seems to define how the _service provider_ commits on _the service usage_ towards the _end user_. According to the [types](https://github.com/ambanum/OpenTermsArchive/blob/master/src/app/types.json#L2-L8), this seems to be a `Terms of Services` document type 🙂

**Comment by clementbiron** [comment]
Thanks to you Matti i just added the PubMatic `Terms of Services`document type here : https://github.com/ambanum/OpenTermsArchive/commit/47bd0188a431505ea31f387e8b34b20dbc712fd3
